<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginPage</class>
 <widget class="QDialog" name="loginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>loginPage</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(86, 98, 112);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>311</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="nomLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 243);</string>
      </property>
      <property name="text">
       <string>Nom :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nomEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="passwordLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 243);</string>
      </property>
      <property name="text">
       <string>Mot de passe :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwdEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="message">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelIcon">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>211</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 243);</string>
   </property>
   <property name="text">
    <string>Veuillez vous identifier</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>561</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="loginBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>S'identifier</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
